packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -179,24 +179,66 @@
         <div class="col-md-4">
             <oui-tile data-heading="{{:: 'pci_projects_project_instances_instance_networks_title' | translate }}"
                 data-loading="$ctrl.loading.instance">
-                <oui-tile-definition data-term="{{:: 'pci_projects_project_instances_instance_ipv4_label' | translate }}">
-                    <oui-tile-description>
-                        <oui-clipboard
-                            data-ng-if="$ctrl.instance.hasPublicIpV4()"
-                            data-model="$ctrl.instance.publicIpV4[0].ip|| ''">
-                        </oui-clipboard>
-                        <span data-ng-if="!$ctrl.instance.hasPublicIpV4()">-</span>
-                    </oui-tile-description>
-                </oui-tile-definition>
-                <oui-tile-definition data-term="{{:: 'pci_projects_project_instances_instance_ipv6_label' | translate }}">
-                    <oui-tile-description>
-                        <oui-clipboard
-                            data-ng-if="$ctrl.instance.hasPublicIpV6()"
-                            data-model="$ctrl.instance.publicIpV6[0].ip|| ''">
-                        </oui-clipboard>
-                        <span data-ng-if="!$ctrl.instance.hasPublicIpV6()">-</span>
-                    </oui-tile-description>
-                </oui-tile-definition>
+                <div class="oui-tile__item">
+                    <dl class="oui-tile__definition">
+                        <dt class="oui-tile__term" data-translate="pci_projects_project_instances_instance_ipv4_label"></dt>
+                        <dd class="oui-tile__description">
+                            <oui-clipboard  data-ng-if="$ctrl.instance.hasPublicIpV4()" data-model="$ctrl.instance.publicIpV4[0].ip|| ''"></oui-clipboard>
+                            <span data-ng-if="!$ctrl.instance.hasPublicIpV4()">-</span>
+                        </dd>
+                        <dt class="oui-tile__term" data-translate="pci_projects_project_instances_instance_reverse_ip_label"></dt>
+                        <dd class="oui-tile__description">
+                            <oui-clipboard data-ng-if="$ctrl.instance.ipReverse" data-model="$ctrl.instance.ipReverse.reverse"></oui-clipboard>
+                            <span data-ng-if="!$ctrl.instance.ipReverse">-</span>
+                        </dd>
+                    </dl>
+                    <div class="oui-tile__actions">
+                        <oui-action-menu data-compact data-align="end">
+                            <oui-action-menu-item text="{{:: 'pci_projects_project_instances_instance_ip_reverse_change_label' | translate }}"
```

Comment:
```suggestion
                            <oui-action-menu-item data-text="{{:: 'pci_projects_project_instances_instance_ip_reverse_change_label' | translate }}"
```
but you should use transclusion as `text` is deprecated :)

https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu